### ISSUE TITLE
Midterm Review

### ISSUE THREAD

**Issue by PatNicholson** [andrewkozma/4741-Project#14]
The focus of this project is on using various characteristics of shots in NBA games to predict whether the shot will be made or not. Some characteristics / featuers included are the shot clock, number of dribbles before the shot, time holding the ball before the shot, distance of the shot, etc. 

What’s good:
Your approach for creating a test set from every fifth game seems very smart, as this way you can cover a variety of games at all different points in the season in both the test set and training set. Also, recognizing some of the corrupt data, such as 3-pointers within 10 feet, will definitely help you avoid fitting models than turn out to be incorrect. Using only the uncorrupted data will certainly help you get better results.

Additionally, your visualizations and explanations for the 3 models you’ve fit so far are excellent. You do a great job both illustrating why you fit a model a certain way quantitatively, and then furthermore what the intuition is behind what the model tells you about the data. The Al Jefferson plot was especially interesting, as it showcases some significant differences between models looking for more localized trends (like autoregressive) versus those looking for more global trends.

Suggestions:
I think, in general, you might need to find a clearer objective for this project. From this report, it seems as if you’re just fitting a bunch of different models to predict shooting percentage, and these models with only one feature variable give results that are kind of obvious intuitively (at least for anyone who watches or plays basketball). Perhaps shifting away from just predicting shooting percentage, and trying to predict something more results-oriented, could yield more focused and interesting results. For example, you could look into whether shot selection affects game outcome, and from that find what the ideal distribution of shots is for winning.

Regardless of what you end up focusing on, I think the approach of fitting smaller models to avoid over-fitting might not be the best way to go. Certainly you can try to minimize features, but you can still fit models on more than one feature. It might be interesting to employ ridge regression or lasso in order to see which features are actually significant in predicting certain values, like shooting percentage.
